src/secure_random.cr

```diff
@@ -29,6 +70,14 @@ module SecureRandom
     slice
   end
 
+  # Generates a UUID (Universally Unique Identifier)
+  # 
+  # It includes 32 random hexadecimal characters chained by 4 "-".
+  # The returned string may contain a-z, 0-9.
```

Comment:
Same as `#hex`, this is a hexstring (so `[0-9a-f]`), and more specifically the format is a UUID v4. See https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29